src/components/wishlist/Wishlist.js

```diff
@@ -0,0 +1,39 @@
+import { useData } from "../../Context/stateManage/state-context";
```

Comment:
Suggestion: Add the scroll to top functionality on navigating to product listing page or any other page for that matter.